inc/osvr/ClientKit/ContextC.h

```diff
@@ -49,7 +49,15 @@ OSVR_EXTERN_C_BEGIN
 
 /** @addtogroup ClientKit
     @{
-*/
+    */
+
+/** @brief Ensures the OSVR server process/thread is running. Call once per process.
+ */
+ OSVR_CLIENTKIT_EXPORT OSVR_ReturnCode osvrServerAutoStart();
```

Comment:
I think I will remove the return code altogether. It's too confusing. Right now it will only fail if it can't find the server executable or can't launch it for some reason. But a failure should actually be ignored, because the user may have the server running already and not have the environment variable defined. So you'd have both false positives and false negatives. It should just make no promises about success or failure.